releasenotes/notes/fix-target-dt-4d306f1e9b07f819.yaml

```
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Fixed an issue with test util :func:`~qiskit.test.mock.utils.convert_to_target`
+    in which ``dt`` was not being converted to seconds when reading from parameter
+    ``conf_dict``.
```

We should make this more about the user facing fix and not the mechanics of the fix. Something like:

```suggestion
    Fixed an issue with :class:`~.BackendV2` based fake backend classes from the 
    ``qiskit.providers.fake_provider`` module such as ``FakeMontrealV2`` where the
    value for the :attr:`~.BackendV2.dt` attribute (and the :attr:`.Target.dt` attribute) 
    were not properly being converted to seconds. This would cause issues when
    using these fake backends with scheduling.
```